addons/web/static/src/js/views/kanban/kanban_record.js

```diff
@@ -316,8 +317,23 @@ var KanbanRecord = Widget.extend({
             attrs[key] = value;
         });
         var options = _.extend({}, this.options, { attrs: attrs });
-        var widget = new Widget(this, field_name, this.state, options);
-        var def = widget.replace($field);
+        let widget;
+        let def;
+        if (utils.isComponent(Widget)) {
+            widget = new FieldWrapper(this, Widget, {
+                fieldName: field_name,
+                record: this.state,
+                options: options,
+            });
+            def = widget.mount(document.createDocumentFragment())
+                .then(() => {
+                    $field.replaceWith(widget.$el);
+                    return Promise.resolve();
```

Comment:
not necessary

Reply:
I think you can use mount with options target ($field) and position (self), so that you wouldn't have to move it afterwards. To check...

Reply:
No, I first use mount with position self and owl threw an error:
`Cannot attach 'FieldWrapper' to target node (not same tag name)`
https://github.com/odoo/owl/blob/master/src/component/fiber.ts#L225
It tries to replace \<field> by \<div>

Reply:
ho :'( ok

Reply:
But Promise.resolve() is still not necessary